dataverse-webapp/pom.xml

```diff
@@ -146,11 +146,13 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
+            <version>2.5.1</version>
```

Comment:
We're managing `Jackson` version in base `pom.xml` using BOM in version `2.9.6`. Why do you try to override to a different version here?

Reply:
There was no BOM version in `2.5.1` but after testing with different versions now I can see that `2.8.3` is also not causing the crash.